contributing.md

```diff
@@ -1,26 +1,35 @@
 
 # For developers
 * [Install tabcmd](#install-tabcmd)
-* [Run tabcmd](#run-tabcmd)
-* [Development commands](#development-commands)
 * [Contributing](#contributing)
-* [To add a new command](#to-add-a-new-command)
-* [Why Python\?](#why-python)
-* [Project Structure](#project-structure)
-
+* [Development](#development)
+  * [Dev scripts](#dev-scripts) 
+  * [Why Python\?](#why-python)
+  * [Project structure](#project-structure)
+  * [To add a new command](#to-add-a-new-command)
+  * [Localization](#localization)
+* [Releases](#releases)
+  * [Versioning](#versioning)
+  * [Packaging](#packaging)
+
+
+These instructions are for people who want to download the code and edit it directly. If you are interested in tabcmd but not the code, see [here](Readme.md).
+####To work with tabcmd, you need to have **Python 3.7+** installed.
 
 
-## Install tabcmd
-These instructions are only necessary if you want to download the code and run it directly. If you are interested in tabcmd but not the code, see [here](Readme.md).
-####To work with tabcmd, you need to have **Python 3.7+** installed.
+## Contributing
 
-### 
+Code contributions and improvements by the community are welcomed!
 
+See the LICENSE file for current open-source licensing and use information. 
 
+Before we can accept pull requests from contributors, we require a signed [Contributor License Agreement (CLA)](http://tableau.github.io/contributing.html).
+ 
 
 ## Development
 
-To work on the tabcmd code, use these scripts. On Windows, 
```

Comment:
Are we expecting Windows devs to use WSL? Can we convert the Bash scripts under `bin/` to Python or provide a corresponding `.bat` or `.ps1` script?

Reply:
These all run on the Windows command line - all my machines are Windows.